velox/functions/sparksql/tests/ArithmeticTest.cpp

```diff
@@ -224,5 +224,20 @@ TEST_F(CeilFloorTest, Limits) {
       floor<double>(-std::numeric_limits<double>::infinity()));
 }
 
+class BinTest : public SparkFunctionBaseTest {
+ protected:
+  std::optional<std::string> bin(std::optional<std::int64_t> arg) {
+    return evaluateOnce<std::string, int64_t>("bin(c0)", {arg}, {BIGINT()});
+  }
+};
+
+TEST_F(BinTest, bin) {
+  EXPECT_EQ(bin(std::nullopt), std::nullopt);
+  EXPECT_EQ(bin(13), "1101");
+  EXPECT_EQ(
```

Comment:
can you add two more cases, one for 0(all 0s) and another for int64_max (all 1s)

Reply:
Already add these cases, thanks!